src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = 0;
+
+  if (days < 3) {
+    totalCost = days * 40;
+  } else if (days < 7) {
+    totalCost = (days * 40) - 20;
+  } else if (days >= 7) {
```

Comment:
```suggestion
  } else {
```
Если предыдущие два условия не выполнились - очевидно что days >= 7